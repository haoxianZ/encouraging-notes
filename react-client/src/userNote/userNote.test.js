import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Note from './userNote'

describe(`Note component`, () => {

  it('renders a .Note by default', () => {
    const wrapper = shallow(<Note />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  
})
