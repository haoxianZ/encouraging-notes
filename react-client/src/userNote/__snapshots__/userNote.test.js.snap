// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders a .Note by default 1`] = `
<div
  className="Note"
>
  <p
    className="content"
  >
    <br />
  </p>
  <p>
     people 
    <FcLike />
     this note!
  </p>
  <button
    className="Note__delete"
    onClick={[Function]}
    type="button"
  >
    Remove
  </button>
  <button
    className="Note_edit"
  >
    <Link
      className="button"
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/users/undefined/notes/undefined/edit"
    >
       Edit
    </Link>
  </button>
</div>
`;
